mgmt-hub/deploy-mgmt-hub.sh

```diff
@@ -266,19 +303,19 @@ areCmdsInstalled() {
 isDockerComposeAtLeast() {
     : ${1:?}
     local minVersion=$1
-    if ! command -v docker-compose >/dev/null 2>&1; then
+    if ! isCmdInstalled docker-compose; then
         return 1   # it is not even installed
     fi
     # docker-compose is installed, check its version
-    lowerVersion=$(echo -e "$(docker-compose version --short)\n$minVersion" | sort -V | head -n1)
-    if [[ $lowerVersion == $minVersion ]]; then
+    lowerVersion=$(echo -e "$(${DOCKER_COMPOSE_CMD} version --short)\n$minVersion" | sort -V | head -n1)
+    if [[ $lowerVersion == $minVersion ]] || [[ $lowerVersion > $minVersion ]]; then
```

Comment:
Why was the change to this line necessary?

Reply:
1/ I cannot use `docker-compose` syntax directly because on `ppc64le` I use Python-based version of it that has different syntax because of pipenv tool. So I use `DOCKER_COMPOSE_CMD` variable to mask this difference.
2/ Adding the second condition `[[ $lowerVersion > $minVersion ]]` in the function `isDockerComposeAtLeast` I fixed wrong logics because this function has to return` true (0)` when installed version od Docker Compose is equal or more according to comments in the code.

Reply:
No, the logic wasn't wrong. The `sort -V` command sorts the list in ascending order, so what is returned is the lowest version. Since we added our minimum required version to our list, if it is the lowest version, then we are good. Otherwise we have to upgrade docker-compose.

Reply:
If we have installed version e.g. 1.22.0 then non-fixed code will return `false (1)`, you will see message ''docker-compose is not installed or not at least version 1.21.0, installing/upgrading it..." and code will try to install either the latest version in Binary mode  or minimal 1.21.0 in Python-based mode. However we already have satisfied version 1.22.0 in our environment. Is it correct logics? From understanding is not. The fixed code will not try to install any version because we already have a good one.

Reply:
You didn't tell me **why** the current logic isn't working. There are 2 things wrong with your check `$lowerVersion > $minVersion` : 

1. Since `minVersion` is added to the list and the list is sorted to return the lowest version, that lowest version can never by greater than `minVersion`
2. `$lowerVersion > $minVersion` just compares the 2 versions alphabetically, which isn't always correct for all forms of versions. That's exactly why we use `sort -V`

Either show me the output of `echo -e "$(${DOCKER_COMPOSE_CMD} version --short)\n$minVersion" | sort -V` in the case where your check is needed, or remove it.

Reply:
@bmpotter We apologize, original check was correct - Yuri removed that change.